cmd/edgectl/aes_install_errors.go

```diff
@@ -30,9 +32,10 @@ func (i *Installer) EmailRequestError(err error) Result {
 // Unable to get a kubectl path.
 func (i *Installer) NoKubectlError(err error) Result {
 	url := "https://www.getambassador.io/docs/latest/topics/install/help/no-kubectl"
+
 	return Result{
 		Report:       "fail_no_kubectl",
-		ShortMessage: "kubectl not found in $PATH",
+		ShortMessage: "The installer was unable to find kubectl in your $PATH",
```

Comment:
I'm not sure about prefixing messages like this one and others with "The installer was/could/etc." as opposed to just stating the error condition. I'm sure Bjorn will have a preference on the desired wording as part of future improvements but for now we need to ensure that every error path HAS a message...

Reply:
This is what they look like now.

![new AESPodStartupError](https://user-images.githubusercontent.com/635244/78847385-76fb8480-79c3-11ea-9056-ee511fde0ff2.png)

Reply:
let me know if this is still something you want to change.

Reply:
It's fine, was just thinking we'd do the wordsmithing after we address the problems in 1.4.1.